fastestimator/op/numpyop/numpyop.py

```diff
@@ -65,8 +65,8 @@ def forward(self, data: Union[np.ndarray, List[np.ndarray]],
         """
         return data
 
-    def forward_batch(self, data: Union[Tensor, List[Tensor]],
-                      state: Dict[str, Any]) -> Union[np.ndarray, List[np.ndarray]]:
+    def forward_batch(self, data: Union[Tensor, List[Tensor]], state: Dict[str,
+                                                                           Any]) -> Union[np.ndarray, List[np.ndarray]]:
```

Comment:
is this formatting correct?

Reply:
That's how VS Code automatically applied formatting for me.